models/article_model.js

```diff
@@ -1,7 +1,14 @@
 const db = require("../db/connection")
 
-function listArticles() {
-  const queryStr = `
+function listArticles(sort_by = "created_at", order = "desc") {
+  const validOptions = ["author", "topic", "created_at"]
+  const validOrders = ["asc", "desc"]
+
+  if(!validOptions.includes(sort_by) || !validOrders.includes(order)) {
+    return Promise.reject({ status: 400, msg: 'Invalid input' })
+  }
+
+  let queryStr = `
```

Comment:
Very nicely handled using default parameter values and a green list array 🏆 I think if it was my API I would let my clients request to sort by `title`, `votes` and `comment_count` as well 🤔